src/mainboard/clevo/tgl-u/variants/ns50mu/devicetree.cb

```diff
@@ -1,4 +1,21 @@
 chip soc/intel/tigerlake
+# CPU (soc/intel/tigerlake/cpu.c)
+    register "power_limits_config[POWER_LIMITS_U_4_CORE]" = "{
+		.tdp_psyspl2 = 65,
```

Comment:
psysPL1 and psysPL2 work concurrently with PL1 and PL2, we should probably set those as well.

BTW why we do not set psysPL1?

Reply:
PL2 / 3 are programmed to base TDP and base TDP * 125 in by coreboot by default

Reply:
I'm not sure if we need PsysPL1, and what value would make sense here. Intel docs recommend setting it to device cooling capacity.

Reply:
> PL2 / 3 are programmed to base TDP and base TDP * 125 by coreboot by default

Ahh right. 

> I don't believe the RoP power draw contributes significantly to thermals here.

Fine.